dashboard/modules/job/job_consts.py

```diff
@@ -1,2 +1,13 @@
+import os
+
+# Job agent consts
+JOB_DIR = "{temp_dir}/job/{job_id}/"
+JOB_UNPACK_DIR = os.path.join(JOB_DIR, "package")
+JOB_DRIVER_ENTRY_FILE = os.path.join(JOB_DIR, "driver-{uuid}.py")
```

Comment:
```suggestion
JOB_DRIVER_ENTRY_FILE_TEMPLATE = os.path.join(JOB_DIR, "driver-{uuid}.py")
```

Reply:
I think it's not necessary to add a _TEMPLATE suffix to all the const names.